### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by cuauh-cabrera**
### Below are listed all the features included in this PR:

- [ ] Add ESLint.
- [ ] Add a responsive mobile navbar.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -975,34 +1100,39 @@ ol a {
...
-    width: 99%;
+    background-color: rgba(255, 255, 255, 0);
+    width: 100%;
+    height: min-content;
     visibility: visible;
     display: flex;
     flex-direction: row;
     justify-content: center;
+    position: fixed;
   }
```
Comment (AlphaNtihinduka) [inline styles.css:1113]: - [ ] I suggest that you consider that the height of the mobile menu covers 100% height of the screen. This is to ensure that we try as much as possible to stick to the [Figma template design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-468&t=XXaeGqDDiMDPDoAc-0) and have a mobile menu with good user Interface ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @cuauh-cabrera 🙋‍♂️
Good job working on implementing the mobile menu. You have done the design to do the expectation and incredibly well. 👏  

TO HIGHLIGHT:
- The mobile menu appears when user clicks (or taps) the hamburger button.
- On mobile the hamburger menu button is visible.
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

## Required changes :recycle: 
- Kindly refer to the inline review comments ☺️ 

## Optional suggestions
N/A


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -18,20 +18,20 @@ jobs:
...
+  #         node-version: "18.x"
+  #     - name: Setup Webhint
+  #       run: |
+  #         npm install --save-dev hint@7.x
+  #         [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+  #     - name: Webhint Report
+  #       run: npx hint .
   stylelint:
     name: Stylelint
     runs-on: ubuntu-22.04
```
Comment (AbelG101) [inline .github/workflows/linters.yml]: - [ ] [OPTIONAL] The previous reviewer forgot to mention this but please make sure the check for `eslint` is running properly this means there should be 5 checks in total (counting `webhint`). I noticed you forgot to add some lines of code for checking `eslint` in your `linters.yml` file above, I've attached below the code you need for that so please copy it just above line `49`. Please visit this [repo](https://github.com/microverseinc/linters-config/blob/master/html-css-js) for more info. 😉
```yml
eslint:
    name: ESLint
    runs-on: ubuntu-22.04
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version: "18.x"
      - name: Setup ESLint
        run: |
          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.eslintrc.json
      - name: ESLint Report
        run: npx eslint .
```

**Review by AbelG101 — APPROVED** [review]
# Hi @cuauh-cabrera 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Menu opens and closes as expected ✔
- Menu option links redirect to the proper page on click ✔
- Well-structured files ✔

## Optional suggestions
- Check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**